core/src/main/java/com/linecorp/armeria/common/util/TimeoutController.java

```diff
@@ -24,27 +24,38 @@
 public interface TimeoutController {
 
     /**
-     * Initializes the timeout scheduler with the specified {@code timeoutMillis}.
+     * Schedules a new timeout with the specified {@code timeoutMillis}.
+     * If a timeout is scheduled already, this method will not start a new timeout.
+     * @return {@code true} if the timeout is scheduled.
```

Comment:
Could you also document when `false` is returned, e.g. `{@code false} if the timeout could not be scheduled because .... or ....` (Please fix all methods.)